### PR TITLE
Add QuizUp to fastlane examples

### PR DESCRIPTION
**Pull request by alliannas**
Let me just first say, thank you so much for creating this amazing tool. It has saved us countless hours when doing submissions to Apple!

So to give you a little context on why our setup is like its here, is a little explanation on how we do things.
#### For Development

We use a feature branch model and when a feature is merged in (or a part of it, if its large we split it up into multiple PRs) our CI automatically builds the `hourly_build` lane and sends to our internal hockey app.

Each night, if there were any changes done to master we upload a build to our hockey beta group.
This is the `daily_build` lane.
#### Translations

QuizUp is currently supported in 7 languages, and its not possible for us to have a very detailed as many of we the features we do are behind an AB tests. 
Our global champions create in advance a list of funny changelogs for each locale and our CI tool selects 1 of them for each release to the App Store.

To accomplish this we use 2 custom actions as explained below.
#### Release flow

We have two main branches, _release_ and _master_. All work is done against the `master` branch and all app store releases are done on the `release` branch.

While waiting for review we still continue to merge new and potentially untested code into `master`.

To test the App Store build we use Testflight and if a critical bug is found in that build we fix that on the release branch do call the `resubmit`, which only updates the build number.
#### Custom actions

We have a lot of custom actions for things we have to do, like selecting the changelogs, updating translations with our internal tool `uhura` and a few minors ones.

---

All feedback is greatly appreciated and we look forward to hear from you and hope we can provide some value with open sourcing our fastfile.

QuizUp iOS Team.

### CODE REVIEW

File: QuizUp/actions/prepare_change_log.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        localesWithMissingChangeLog = []
+
+        locales.each do |locale|
+            remainingChangeLogsForLocale = self.check_line_count_for_change_log_file(locale)
+            if remainingChangeLogsForLocale == 0
+                localesWithMissingChangeLog.push(locale)
+            end
+        end
+
+        unless localesWithMissingChangeLog.empty?
```
Comment (KrauseFx) [inline QuizUp/actions/prepare_change_log.rb]: Variables shouldn't be camel case, but use `_` 👍

**Review by KrauseFx — COMMENTED** [review]
👍

**Comment by KrauseFx** [comment]
Hey @alliannas, awesome, thanks for preparing this. The PR looks great, however could you update it to use `_` instead of camel case for variable names?

**Comment by alliannas** [comment]
Yeah will do.

File: QuizUp/actions/prepare_change_log.rb (hunk 1 above)
Comment (alliannas) [reply QuizUp/actions/prepare_change_log.rb]: Fixed in b950e34e525b3150b071ddb80436d934b983ed74 :)

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
Cool, thank you so much for contributing this, I'm sure this will help a lot of people 👍

**Pull request merged**